doi-requests/src/main/java/no/unit/nva/doirequest/create/CreateDoiRequestHandler.java

```diff
@@ -33,20 +35,22 @@
 
     public static final String DOI_ALREADY_EXISTS_ERROR = "A DOI request already exists";
     public static final String USER_IS_NOT_OWNER_ERROR = "User does not own the specific publication";
+
+    private static final Clock CLOCK = Clock.systemDefaultZone();
     private final DoiRequestService doiRequestService;
     private final MessageService messageService;
     private final ResourceService resourceService;
 
     @JacocoGenerated
     public CreateDoiRequestHandler() {
-        this(AmazonDynamoDBClientBuilder.defaultClient(), Clock.systemDefaultZone());
+        this(AmazonDynamoDBClientBuilder.defaultClient(), EXTERNAL_SERVICES_HTTP_CLIENT, CLOCK);
```

Comment:
EXTERNAL_SERVICES_HTTP_CLIENT is the HttpClient used for Bare. Since, it is likely to be used for connecting to other services as well, I want to avoid having to create one HttpClient for each service.